examples/slackbot/Notebook.ipynb

```diff
@@ -141,17 +142,15 @@
     "\n",
     "# Format for the OpenAI API\n",
     "def format_prompt(prompt):\n",
-    "    return [f\"{p.user} @ {p.ts} ---> {p.text}\" for p in prompt].join(\"\\n\\n\") + \"\\n\\n###\\n\\n\"\n",
+    "    return \"start -> \" + \"\\n\\n\".join([f' {msg.user} --> {msg.text} ' for msg in prompt]) + \"\\n\\n###\\n\\n\"\n",
     "examples_df.prompt = examples_df.prompt.apply(format_prompt)\n",
     "\n",
     "def format_completion(completion):\n",
-    "    return [le.predict(u) for u in completion].join(\" \") + \"###\"\n",
+    "    return \" \" + (\" \".join([le.transform(u) for u in completion]) if len(completion) > 0 else \"nil\") + \" end\"\n",
     "examples_df.completion = examples_df.completion.apply(format_completion)\n",
     "\n",
-    "# Split training & validation\n",
-    "train, valid = train_test_split(examples_df, test_size=0.2, random_state=42)\n",
-    "train.to_json(\"train.jsonl\", orient='records', lines=True)\n",
-    "valid.to_json(\"valid.jsonl\", orient='records', lines=True)"
+    "# Write examples to file\n",
+    "examples_df.to_json(\"examples.jsonl\", orient='records', lines=True)"
```

Comment:
let the OpenAI tool do the split, since it will do it anyway.  no way to disable it.